src/status_im/contexts/wallet/account/bridge_send/view.cljs

```diff
@@ -11,7 +11,8 @@
   []
   [rn/view {:style style/bridge-send-wrapper}
    [input-amount/view
-    {:button-one-label (i18n/label :t/confirm-bridge)
+    {:stack-id         :wallet-bridge-send
```

Comment:
btw, this `:stack-id` is very confusing. 
It needs to be clearer imo. Is it the "current" stack-id or the "next" stack-id

Reply:
It should be the screen id actually, Ideally we should grab it from the navigation library, but need to do some research how to do it. I'll update to `screen-id` if you are okay with that

Reply:
I think :current-screen-id is more descriptive here so.